projects/components/src/overlay/sheet/sheet-overlay.component.scss

```diff
@@ -62,3 +43,17 @@
     }
   }
 }
+
+.attached-trigger {
+  cursor: pointer;
+  background: white;
+  width: 200px;
+  height: 40px;
+  transform: translate(calc(-50% - 20px), 0px) rotate(270deg);
+  position: relative;
+  top: -50%;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
```
box-shadow: -3px 1px 24px rgba(0, 0, 0, 0.12), -1px -1px 8px rgba(0, 0, 0, 0.08);
clip-path: inset(-20px -20px 0px -20px);
```
We should also add these two properties here as well to have proper box-shadow in non collapsed view. You can try this.

Reply:
added